packages/documentation-site/patternfly-docs/content/get-started/upgrade.md

```diff
@@ -9,25 +9,32 @@ import { Alert } from '@patternfly/react-core';
 <Alert variant="info" isInline title="Before you upgrade"> Make sure that you have already completed [the PatternFly 5 upgrade](https://www.patternfly.org/get-started/upgrade) before attempting to support PatternFly 6. This will ensure that your product has addressed any necessary changes from our previous release.
```

Comment:
Just a note that I'm guessing the v5 upgrade guide URL will need to be updated to reference the v5 site once it goes live?